gix-blame/src/file/function.rs

```diff
@@ -391,6 +414,7 @@ fn blob_changes(
     let number_of_lines_in_destination = input.after.len();
     let change_recorder = ChangeRecorder::new(number_of_lines_in_destination.try_into().unwrap());
 
+    stats.blobs_diffed += 1;
```

Comment:
Minor issue: should this come after `gix_diff::blob::diff`?

Reply:
At this point, the blob-diff cannot fail anymore, and it was convenient to put it there.
I will put it after in one the diff as it may be confusing to see it counted early.